### DISCUSSION TITLE
Feature request of new data source support

### DISCUSSION THREAD

**Discussion by wwwy3y3** [Canner/vulcan-sql#232] [General]
Didn't see your database supported in VulcanSQL. Feel free to let us know here!

We'll adjust our roadmap based on community feedback.

**Comment by wwwy3y3** [comment]
Saw MySQL/MariaDB request on discord.
Upvote if you want to use MySQL/MariaDB with VulcanSQL as well.

**Comment by wwwy3y3** [comment]
Saw Athena/iceberg request from dbt Slack channel.
Upvote if you want to use Athena (not limited to iceberg table format) with VulcanSQL as well.

**Comment by wwwy3y3** [comment]
Met someone at a meetup, asking us if MongoDB is supported.
Upvote if you want to use MongoDB with VulcanSQL.

**Reply by wwwy3y3** [reply]
Does not have SQL interface tho. Could we maybe do it by allowing users to write Mongo [query selectors](https://www.mongodb.com/docs/manual/reference/operator/query/) ?

One issue I could think of is probably jinja compiler confusing `{}` of mongo query selectors with jinja's `{}`

**Comment by cyyeh** [comment]
Saw Trino request on Discord.
Upvote if you want to use Trino with VulcanSQL as well.

**Comment by cyyeh** [comment]
Saw Vertica request on reddit.
Upvote if you want to use Vertica with VulcanSQL as well.

**Comment by cyyeh** [comment]
Redshift
